_submissions/round-12/3/2015-3-28-andrew-macdonald.md

```diff
@@ -0,0 +1,42 @@
+---
+data: '2015-03-28'
+round: Round 12
+title: Demotivation
+author: Andrew MacDonald
+permalink: 2015/3/andrew-macdonald-motivation
+tags: Motivation
```

Comment:
Since tags can be a list, they have to be written like this:

    tags:
      - Motivation